Sources/dump-unified-graph/main.swift

```diff
@@ -0,0 +1,110 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import ArgumentParser
+import SymbolKit
+
+struct DumpUnifiedGraph: ParsableCommand {
```

Comment:
Should we `@main` this and rename the file to `DumpUnifiedGraph`?

```suggestion
@main
struct DumpUnifiedGraph: ParsableCommand {
```